src/main/java/lotto/vo/Elements.java

```diff
@@ -0,0 +1,86 @@
+package lotto.vo;
+
+import lotto.exception.InvalidFormatException;
+import lotto.exception.NegativeElementException;
+import lotto.utils.StringUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Elements {
+
+    private static final String DEFAULT_DELIMITER = "[,:]";
+
+    private final String formula;
+    private final List<Integer> elements;
+
+    public Elements(final String formula) {
+        this.formula = formula;
+        this.elements = parse();
+    }
+
+    private List<Integer> parse() {
+
+        if ( StringUtils.isNullOrEmpty(formula) ) {
+            return new ArrayList<>();
+        }
+
+        return makeList( split(formula) );
+    }
+
+    private String[] split(final String formula) {
+
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(formula);
+        if (m.find()) {
+            String customDelimiter = Pattern.quote(m.group(1));
+            return m.group(2).split(customDelimiter);
+        }
+
+        return formula.split(DEFAULT_DELIMITER);
+    }
+
+    private List<Integer> makeList(final String[] stringElements) {
+        List<Integer> elements = new ArrayList<>();
+        for (String elm : stringElements) {
+            validateElementNegative(validateElementNumeric(elm));
+            elements.add(Integer.parseInt(elm));
+        }
+        return elements;
+    }
+
+    private Integer validateElementNumeric(final String item) {
```

Comment:
참조 자료형 대신 기본 자료형을 사용하면 어떨까요? 아래의 글을 참고하세요.
https://jaehun2841.github.io/2019/03/01/effective-jave-item61

```suggestion
    private int validateElementNumeric(final String item) {
```